python/create_fake_data.py

```diff
@@ -1,60 +1,68 @@
+"""Creates the fake data to be used by the fake backend."""
 import json
```

Comment:
This file is curenttly hard to read. Can we have a main(), a method to create a random_release( with args) and a method to create a set of release with args. That way we can re-use this in tests.